simulariumio/trajectory_converter.py

```diff
@@ -507,40 +139,35 @@ def to_JSON(self):
         Return the current simularium data in JSON format
 
         """
```

Comment:
updated these functions since they've been moved to the writers. `TrajectoryConverter.write_external_JSON()` is now `JsonWriter.save()`